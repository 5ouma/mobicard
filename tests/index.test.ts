import { experimental_AstroContainer as AstroContainer } from "astro/container";
import { expect, test } from "vitest";

import Index from "../src/pages/index.astro";

test("Index", async () => {
  const container: AstroContainer = await AstroContainer.create();
  const result: string = await container.renderToString(Index);

  expect(result).toMatchSnapshot();
});
