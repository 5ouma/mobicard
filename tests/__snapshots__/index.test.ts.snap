// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Index 1`] = `
"<!---

  ðŸ“‡ Mobicard created by @5ouma
  https://github.com/5ouma/mobicard

  - ðŸš€ Built with Astro
  - âš¡ Tested with Vitest
  - ðŸŒ± Using LINE Seed
  - ðŸ‘‘ Using MingCute Icon

--><html lang="en" data-astro-cid-sckkx6r4> <head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, viewport-fit=cover"><title>Mobicard</title><meta name="description" content="ðŸ“‡ Turn your smartphone into a name card"><link rel="icon" href="https://images.5ouma.me/avatar/default.png"><link rel="apple-touch-icon" href="https://images.5ouma.me/avatar/default.png"><meta property="og:title" content="Mobicard"><meta property="og:description" content="ðŸ“‡ Turn your smartphone into a name card"><meta property="og:image" content="https://images.5ouma.me/avatar/default.png"><meta name="twitter:card" content="summary"><meta name="twitter:title" content="Mobicard"><meta name="twitter:description" content="ðŸ“‡ Turn your smartphone into a name card"><meta name="twitter:image" content="https://images.5ouma.me/avatar/default.png"><meta name="theme-color" content="#f4f4f4" media="(prefers-color-scheme: light)"><meta name="theme-color" content="#242424" media="(prefers-color-scheme: dark)"></head> <body data-astro-cid-sckkx6r4>  <main data-astro-cid-j7pv25f6> <div id="info" data-astro-cid-j7pv25f6> <div id="bio" data-astro-cid-yljsttrw> <div id="profile" data-astro-cid-yljsttrw> <picture data-astro-cid-yljsttrw="true"> <source srcset="/_image?href=https%3A%2F%2Fimages.5ouma.me%2Favatar%2Fdefault.png&#38;w=4000&#38;h=4000&#38;f=avif" type="image/avif"><source srcset="/_image?href=https%3A%2F%2Fimages.5ouma.me%2Favatar%2Fdefault.png&#38;w=4000&#38;h=4000&#38;f=webp" type="image/webp"><source srcset="/_image?href=https%3A%2F%2Fimages.5ouma.me%2Favatar%2Fdefault.png&#38;w=4000&#38;h=4000&#38;f=jpg" type="image/jpeg">  <img src="/_image?href=https%3A%2F%2Fimages.5ouma.me%2Favatar%2Fdefault.png&#38;w=4000&#38;h=4000&#38;f=png" alt="Mobicard's avatar" data-astro-cid-yljsttrw="true" width="4000" height="4000" loading="lazy" decoding="async"> </picture> <h1 data-astro-cid-yljsttrw>Mobicard</h1> </div> <p data-astro-cid-yljsttrw>ðŸ“‡ Turn your smartphone into a name card</p> </div>  <div id="contacts" data-astro-cid-j7pv25f6> <div data-astro-cid-j4povljh> <a href="https://github.com/5ouma/mobicard" target="_blank" rel="noopener noreferrer" aria-label="Visit github.com" data-astro-cid-j4povljh> <svg width="1em" height="1em" data-astro-cid-j4povljh="true" data-icon="mingcute:home-3-line">   <symbol id="ai:mingcute:home-3-line" viewBox="0 0 24 24"><g fill="none"><path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M10.772 2.688a2 2 0 0 1 2.317-.099l.139.1l8.384 6.52c.721.561.37 1.692-.499 1.785l-.116.006H20v8a2 2 0 0 1-1.85 1.995L18 21H6a2 2 0 0 1-1.994-1.85L4 19v-8h-.997C2.09 11 1.671 9.892 2.3 9.285l.088-.076zM12 4.267L5.625 9.225c.229.185.375.468.375.785V19h3v-5a3 3 0 1 1 6 0v5h3v-8.99c0-.317.146-.6.375-.785zM12 13a1 1 0 0 0-1 1v5h2v-5a1 1 0 0 0-1-1"/></g></symbol><use href="#ai:mingcute:home-3-line"></use>  </svg> <span aria-hidden="true" data-astro-cid-j4povljh> github.com/5ouma/mobicard </span> </a> <a href="mailto:email@example.com" aria-label="Send an email to email@example.com" data-astro-cid-j4povljh> <svg width="1em" height="1em" data-astro-cid-j4povljh="true" data-icon="mingcute:mail-line">   <symbol id="ai:mingcute:mail-line" viewBox="0 0 24 24"><g fill="none"><path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M20 4a2 2 0 0 1 1.995 1.85L22 6v12a2 2 0 0 1-1.85 1.995L20 20H4a2 2 0 0 1-1.995-1.85L2 18V6a2 2 0 0 1 1.85-1.995L4 4zm0 3.414l-6.94 6.94a1.5 1.5 0 0 1-2.12 0L4 7.414V18h16zM18.586 6H5.414L12 12.586z"/></g></symbol><use href="#ai:mingcute:mail-line"></use>  </svg> <span aria-hidden="true" data-astro-cid-j4povljh>email@example.com</span> </a> </div>  <ul aria-label="Contacts" data-astro-cid-j7pv25f6> <li data-astro-cid-j7pv25f6> <a href="https://bsky.app/profile/bsky.app" target="_blank" rel="noopener noreferrer" aria-label="Go to bsky.app's profile on Bluesky" data-astro-cid-ilwnnabb style="--accent-color: #0285FF;"> <svg width="1em" height="1em" data-astro-cid-ilwnnabb="true" data-icon="mingcute:bluesky-social-line">   <symbol id="ai:mingcute:bluesky-social-line" viewBox="0 0 24 24"><g fill="none" fill-rule="evenodd"><path d="m12.594 23.258l-.012.002l-.071.035l-.02.004l-.014-.004l-.071-.036q-.016-.004-.024.006l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.016-.018m.264-.113l-.014.002l-.184.093l-.01.01l-.003.011l.018.43l.005.012l.008.008l.201.092q.019.005.029-.008l.004-.014l-.034-.614q-.005-.019-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.003-.011l.018-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M4.747 5.105a.94.94 0 0 0-.41.892l.314 3.287A3 3 0 0 0 7.637 12H9a1 1 0 0 1 .371 1.928l-2.34.937c-.885.354-1.122 1.32-.664 1.932c.288.384.576.731.84.996c.233.233.532.485.865.74c.619.475 1.625.214 1.94-.734l1.04-3.115a1 1 0 0 1 1.897 0l1.038 3.115c.316.948 1.322 1.21 1.941.735a9 9 0 0 0 .865-.741c.265-.265.553-.612.84-.996c.458-.612.222-1.578-.664-1.932l-2.34-.937A1 1 0 0 1 15 12h1.363a3 3 0 0 0 2.986-2.716l.314-3.287a.94.94 0 0 0-.41-.892a.75.75 0 0 0-.858-.001c-2.645 1.708-4.05 3.442-5.5 6.343a1 1 0 0 1-1.79 0c-1.45-2.901-2.855-4.635-5.5-6.343a.75.75 0 0 0-.858.001m-2.4 1.082C2.13 3.913 4.567 2.053 6.69 3.424C9.13 5 10.662 6.639 12 8.9c1.338-2.261 2.87-3.9 5.31-5.476c2.123-1.371 4.56.49 4.344 2.763l-.314 3.287a5 5 0 0 1-2.783 4.02c1.329 1.039 1.818 2.978.677 4.502c-.317.423-.669.853-1.027 1.211c-.317.317-.69.628-1.062.914c-1.88 1.441-4.375.35-5.055-1.69l-.09-.269l-.09.27c-.68 2.04-3.175 3.13-5.055 1.689a11 11 0 0 1-1.062-.914a11.5 11.5 0 0 1-1.027-1.211c-1.141-1.524-.651-3.463.678-4.502a5 5 0 0 1-2.784-4.02z"/></g></symbol><use href="#ai:mingcute:bluesky-social-line"></use>  </svg> <span data-astro-cid-ilwnnabb style="--accent-color: #0285FF;">bsky.app</span> </a>  </li><li data-astro-cid-j7pv25f6> <a href="https://github.com/github" target="_blank" rel="noopener noreferrer" aria-label="Go to github's profile on GitHub" data-astro-cid-ilwnnabb style="--accent-color: #181717;"> <svg width="1em" height="1em" data-astro-cid-ilwnnabb="true" data-icon="mingcute:github-line">   <symbol id="ai:mingcute:github-line" viewBox="0 0 24 24"><g fill="none"><path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M6.315 6.176c-.25-.638-.24-1.367-.129-2.034a6.8 6.8 0 0 1 2.12 1.07c.28.214.647.283.989.18A9.3 9.3 0 0 1 12 5c.961 0 1.874.14 2.703.391c.342.104.709.034.988-.18a6.8 6.8 0 0 1 2.119-1.07c.111.667.12 1.396-.128 2.033c-.15.384-.075.826.208 1.14C18.614 8.117 19 9.04 19 10c0 2.114-1.97 4.187-5.134 4.818c-.792.158-1.101 1.155-.495 1.726c.389.366.629.882.629 1.456v3a1 1 0 0 0 2 0v-3c0-.57-.12-1.112-.334-1.603C18.683 15.35 21 12.993 21 10c0-1.347-.484-2.585-1.287-3.622c.21-.82.191-1.646.111-2.28c-.071-.568-.17-1.312-.57-1.756c-.595-.659-1.58-.271-2.28-.032a9 9 0 0 0-2.125 1.045A11.4 11.4 0 0 0 12 3c-.994 0-1.953.125-2.851.356a9 9 0 0 0-2.125-1.045c-.7-.24-1.686-.628-2.281.031c-.408.452-.493 1.137-.566 1.719l-.005.038c-.08.635-.098 1.462.112 2.283C3.484 7.418 3 8.654 3 10c0 2.992 2.317 5.35 5.334 6.397A4 4 0 0 0 8 17.98l-.168.034c-.717.099-1.176.01-1.488-.122c-.76-.322-1.152-1.133-1.63-1.753c-.298-.385-.732-.866-1.398-1.088a1 1 0 0 0-.632 1.898c.558.186.944 1.142 1.298 1.566c.373.448.869.916 1.58 1.218c.682.29 1.483.393 2.438.276V21a1 1 0 0 0 2 0v-3c0-.574.24-1.09.629-1.456c.607-.572.297-1.568-.495-1.726C6.969 14.187 5 12.114 5 10c0-.958.385-1.881 1.108-2.684c.283-.314.357-.756.207-1.14"/></g></symbol><use href="#ai:mingcute:github-line"></use>  </svg> <span data-astro-cid-ilwnnabb style="--accent-color: #181717;">github</span> </a>  </li><li data-astro-cid-j7pv25f6> <a href="https://www.instagram.com/instagram" target="_blank" rel="noopener noreferrer" aria-label="Go to instagram's profile on Instagram" data-astro-cid-ilwnnabb style="--accent-color: #E4405F;"> <svg width="1em" height="1em" data-astro-cid-ilwnnabb="true" data-icon="mingcute:ins-line">   <symbol id="ai:mingcute:ins-line" viewBox="0 0 24 24"><g fill="none" fill-rule="evenodd"><path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M16 3a5 5 0 0 1 5 5v8a5 5 0 0 1-5 5H8a5 5 0 0 1-5-5V8a5 5 0 0 1 5-5zm0 2H8a3 3 0 0 0-3 3v8a3 3 0 0 0 3 3h8a3 3 0 0 0 3-3V8a3 3 0 0 0-3-3m-4 3a4 4 0 1 1 0 8a4 4 0 0 1 0-8m0 2a2 2 0 1 0 0 4a2 2 0 0 0 0-4m4.5-3.5a1 1 0 1 1 0 2a1 1 0 0 1 0-2"/></g></symbol><use href="#ai:mingcute:ins-line"></use>  </svg> <span data-astro-cid-ilwnnabb style="--accent-color: #E4405F;">instagram</span> </a>  </li><li data-astro-cid-j7pv25f6> <a href="https://mastodon.social/@Mastodon" target="_blank" rel="noopener noreferrer" aria-label="Go to @Mastodon@mastodon.social's profile on Mastodon" data-astro-cid-ilwnnabb style="--accent-color: #6364FF;"> <svg width="1em" height="1em" data-astro-cid-ilwnnabb="true" data-icon="mingcute:mastodon-line">   <symbol id="ai:mingcute:mastodon-line" viewBox="0 0 24 24"><g fill="none"><path d="m12.594 23.258l-.012.002l-.071.035l-.02.004l-.014-.004l-.071-.036q-.016-.004-.024.006l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.016-.018m.264-.113l-.014.002l-.184.093l-.01.01l-.003.011l.018.43l.005.012l.008.008l.201.092q.019.005.029-.008l.004-.014l-.034-.614q-.005-.019-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.003-.011l.018-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M16.432 2.415a5.11 5.11 0 0 1 4.163 4.142c.431 2.361.528 4.682.04 7.108c-.406 2.023-2.006 3.532-4.01 3.893c-1.55.278-3.105.424-4.667.42c.722.144 1.471.17 2.21.186l.632.012l.395.002c.536.007 1.097.048 1.533.34c.266.178.535.51.533.985c0 .429-.231.732-.364.878c-.468.513-1.19.784-1.832.994c-1.042.339-2.46.6-4.06.607c-3.428.016-6.644-2.263-7.478-5.75c-.749-3.133-.743-6.648-.145-9.781a5.03 5.03 0 0 1 4.094-4.02a25.9 25.9 0 0 1 8.956-.016M7.822 4.4a3.03 3.03 0 0 0-2.475 2.424c-.552 2.89-.55 6.117.125 8.942c.602 2.514 2.951 4.227 5.523 4.215q.351-.001.684-.02a5.86 5.86 0 0 1-2.959-1.498c-.519-.495-1.115-1.254-.791-2.014c.201-.471.702-.721 1.2-.657c2.383.31 4.759.224 7.142-.204c1.223-.22 2.165-1.127 2.404-2.318c.426-2.119.35-4.177-.047-6.355a3.11 3.11 0 0 0-2.537-2.53a23.9 23.9 0 0 0-8.269.015M14 6a3 3 0 0 1 3 3v4a1 1 0 1 1-2 0V9a1 1 0 1 0-2 0v3a1 1 0 1 1-2 0V9a1 1 0 1 0-2 0v4a1 1 0 1 1-2 0V9a3 3 0 0 1 5-2.236A3 3 0 0 1 14 6"/></g></symbol><use href="#ai:mingcute:mastodon-line"></use>  </svg> <span data-astro-cid-ilwnnabb style="--accent-color: #6364FF;">@Mastodon@mastodon.social</span> </a>  </li><li data-astro-cid-j7pv25f6> <a href="https://twitter.com/twitter" target="_blank" rel="noopener noreferrer" aria-label="Go to twitter's profile on Twitter" data-astro-cid-ilwnnabb style="--accent-color: #1D9BF0;"> <svg width="1em" height="1em" data-astro-cid-ilwnnabb="true" data-icon="mingcute:twitter-line">   <symbol id="ai:mingcute:twitter-line" viewBox="0 0 24 24"><g fill="none" fill-rule="evenodd"><path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M17.817 6.989c-.353-.508-1.297-1.222-2.826-.914c-.892.179-1.425.62-1.764 1.202c-.361.62-.536 1.463-.536 2.444a1 1 0 0 1-1 1c-2.366 0-4.618-.703-6.513-2.604a10.5 10.5 0 0 0-.168 2.34c.053 1.086.312 2.175.927 3.108c.605.917 1.6 1.757 3.264 2.285a1 1 0 0 1 .41 1.655a8.5 8.5 0 0 1-1.796 1.405c1.06.11 2.082.118 3.03.032c1.92-.174 3.449-.723 4.405-1.5c2.013-1.632 3.09-4.066 2.89-8.144c-.034-.67.613-1.393.925-1.942c-.471.088-.943.072-1.248-.367M4.594 4.984a1 1 0 0 1 .941.429C7.011 7.572 8.783 8.47 10.75 8.674c.096-.841.323-1.672.75-2.404c.626-1.074 1.644-1.864 3.098-2.156c2.01-.404 3.54.324 4.427 1.215l1.792-.335a1 1 0 0 1 1.053 1.478l-1.72 3.022c.157 4.361-1.055 7.405-3.639 9.502c-1.37 1.112-3.332 1.743-5.485 1.938c-2.17.196-4.623-.041-7.061-.753a1 1 0 0 1 .007-1.922c1.226-.349 2.16-.65 3.003-1.177c-1.199-.636-2.082-1.468-2.707-2.416c-.868-1.318-1.19-2.788-1.254-4.113S3.141 8 3.343 7.115c.115-.505.249-1.011.434-1.495a1 1 0 0 1 .818-.636Z"/></g></symbol><use href="#ai:mingcute:twitter-line"></use>  </svg> <span data-astro-cid-ilwnnabb style="--accent-color: #1D9BF0;">twitter</span> </a>  </li> </ul> </div> </div> <button id="card" type="button" aria-label="Share your card" data-astro-cid-nzteqis6> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 29 29" shape-rendering="crispEdges" id="qr" data-astro-cid-nzteqis6="true"><path stroke="#000000" d="M0 0.5h7m4 0h2m2 0h5m2 0h7M0 1.5h1m5 0h1m2 0h1m1 0h1m5 0h1m2 0h1m1 0h1m5 0h1M0 2.5h1m1 0h3m1 0h1m1 0h4m2 0h1m1 0h1m1 0h2m2 0h1m1 0h3m1 0h1M0 3.5h1m1 0h3m1 0h1m1 0h1m7 0h2m2 0h1m1 0h1m1 0h3m1 0h1M0 4.5h1m1 0h3m1 0h1m1 0h1m2 0h5m2 0h3m1 0h1m1 0h3m1 0h1M0 5.5h1m5 0h1m1 0h2m1 0h6m3 0h1m1 0h1m5 0h1M0 6.5h7m1 0h1m1 0h1m1 0h1m1 0h1m1 0h1m1 0h1m1 0h1m1 0h7M8 7.5h2m2 0h2m1 0h1m2 0h1m1 0h1M0 8.5h1m1 0h5m7 0h2m1 0h1m4 0h5M0 9.5h1m4 0h1m2 0h4m3 0h2m1 0h7m3 0h1M1 10.5h1m2 0h1m1 0h1m3 0h2m8 0h2m1 0h2M4 11.5h1m2 0h1m3 0h2m1 0h1m1 0h1m2 0h2m1 0h1m2 0h1m1 0h1M0 12.5h4m1 0h2m1 0h2m6 0h2m7 0h2M2 13.5h1m1 0h2m2 0h2m1 0h1m1 0h3m3 0h6m3 0h1M0 14.5h4m2 0h3m1 0h1m2 0h5m2 0h2m3 0h2M0 15.5h1m1 0h1m1 0h2m6 0h2m1 0h1m2 0h1m5 0h1m2 0h1M1 16.5h1m3 0h2m1 0h2m1 0h1m2 0h2m1 0h2m6 0h2M0 17.5h1m1 0h1m2 0h1m1 0h1m1 0h2m1 0h1m2 0h6m1 0h3m1 0h1m1 0h1M0 18.5h1m2 0h1m1 0h2m2 0h1m1 0h2m4 0h1m3 0h1m1 0h1m2 0h1M0 19.5h1m3 0h1m3 0h2m2 0h1m1 0h1m1 0h1m4 0h3m3 0h1M0 20.5h1m2 0h1m2 0h4m1 0h2m3 0h1m1 0h1m1 0h5m1 0h3M8 21.5h8m1 0h4m3 0h5M0 22.5h7m2 0h3m1 0h5m1 0h2m1 0h1m1 0h3M0 23.5h1m5 0h1m1 0h1m1 0h2m1 0h1m1 0h1m1 0h4m3 0h1m3 0h1M0 24.5h1m1 0h3m1 0h1m1 0h1m5 0h2m4 0h5m1 0h3M0 25.5h1m1 0h3m1 0h1m1 0h2m1 0h1m1 0h1m1 0h2m1 0h1m1 0h1m4 0h2M0 26.5h1m1 0h3m1 0h1m1 0h1m1 0h2m1 0h1m4 0h10M0 27.5h1m5 0h1m2 0h1m3 0h1m2 0h1m3 0h1m2 0h3m1 0h1M0 28.5h7m1 0h3m1 0h1m3 0h1m1 0h1m2 0h1m1 0h1m2 0h1"></path></svg> <div data-astro-cid-nzteqis6> <svg width="1em" height="1em" class="icon" id="idcard" data-astro-cid-nzteqis6="true" data-icon="mingcute:idcard-line">   <symbol id="ai:mingcute:idcard-line" viewBox="0 0 24 24"><g fill="none" fill-rule="evenodd"><path d="m12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.018-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M4 4a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V6a2 2 0 0 0-2-2zm0 2h16v12H4zm6.25 3.75a1.75 1.75 0 1 1-3.5 0a1.75 1.75 0 0 1 3.5 0M5 14.5A2.5 2.5 0 0 1 7.5 12h2a2.5 2.5 0 0 1 2.5 2.5v.5a1 1 0 1 1-2 0v-.5a.5.5 0 0 0-.5-.5h-2a.5.5 0 0 0-.5.5v.5a1 1 0 1 1-2 0zm8-3.5a1 1 0 0 1 1-1h2a1 1 0 1 1 0 2h-2a1 1 0 0 1-1-1m1 2a1 1 0 1 0 0 2h4a1 1 0 1 0 0-2z"/></g></symbol><use href="#ai:mingcute:idcard-line"></use>  </svg> <svg width="1em" height="1em" class="icon" id="check" display="none" data-astro-cid-nzteqis6="true" data-icon="mingcute:checkbox-line">   <symbol id="ai:mingcute:checkbox-line" viewBox="0 0 24 24"><g fill="none" fill-rule="evenodd"><path d="m12.594 23.258l-.012.002l-.071.035l-.02.004l-.014-.004l-.071-.036q-.016-.004-.024.006l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.016-.018m.264-.113l-.014.002l-.184.093l-.01.01l-.003.011l.018.43l.005.012l.008.008l.201.092q.019.005.029-.008l.004-.014l-.034-.614q-.005-.019-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.003-.011l.018-.43l-.003-.012l-.01-.01z"/><path fill="currentColor" d="M5 3a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14a2 2 0 0 0 2-2V5a2 2 0 0 0-2-2zm0 2h14v14H5zm11.95 4.795a1 1 0 1 0-1.414-1.414l-4.95 4.95l-2.121-2.121a1 1 0 0 0-1.415 1.414l2.758 2.758a1.1 1.1 0 0 0 1.556 0z"/></g></symbol><use href="#ai:mingcute:checkbox-line"></use>  </svg> <span aria-hidden="true" data-astro-cid-nzteqis6>mobicard.pages.dev</span> </div> </button>  <script>
  const shareData = {
    title: document.title,
    text: document.head.querySelector('[name="description"]').content,
    url: location.href,
  };

  document.getElementById("card").addEventListener("click", async () => {
    if (
      /iPhone|iPad|Android/i.test(navigator.userAgent) &&
      navigator.canShare(shareData)
    ) {
      try {
        await navigator.share(shareData);
      } catch (error) {
        if (error.name !== "AbortError") console.error("Error sharing:", error);
      }
    } else {
      await navigator.clipboard.writeText(shareData.url);
      const [cardIcon, checkIcon] = [
        document.getElementById("idcard"),
        document.getElementById("check"),
      ];
      [cardIcon.style.display, checkIcon.style.display] = ["none", "block"];
      setTimeout(() => {
        [cardIcon.style.display, checkIcon.style.display] = ["block", "none"];
      }, 1000);
    }
  });
</script> </main>   </body></html> "
`;
